package.json

```diff
@@ -1,6 +1,15 @@
 {
   "name": "mathjax-angular-project",
   "version": "1.1.0",
+  "description": "A quick way to implement Mathjax v3 into your Angular project!",
```

Comment:
update in both readme,nd also

Reply:
? what should i do in readme

Reply:
so no update required in readme.md ?

Reply:
hopefully all cahnges that you did in this package.json would be done in package.json inside mathjax-lib folder also

Reply:
Ok check again